semcore/wizard/src/Wizard.jsx

```diff
@@ -124,9 +171,11 @@ function Stepper(props) {
   return sstyled(styles)(
     <SStepper
       render={Box}
-      role='menuitem'
+      role='tab'
```

Comment:
@msereniti after discussion with Glafira, it seems we have to add navigation with Arrows.
In this case it should be Up/Down Arrow.
But IMO the tabs shouldn't activate on selection, i.e. user has to choose with an Arrow and then confirm with Enter/Space (like in Tabline/Tabpanel with behavior=manual).